test/strategy/EnrichClientFromRemoteConfigStrategyTest.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+var stormpath = require('stormpath');
+
+var common = require('../common');
+var _ = common._;
```

Comment:
Why don't you require the things you need yourself, instead of going through `common`?